apps/common/db/models.py

```diff
@@ -83,7 +83,7 @@ def CASCADE_SIGNAL_SKIP(collector, field, sub_objs, using):
     # 级联删除时，操作日志标记不保存，以免用户混淆
     try:
         for obj in sub_objs:
-            setattr(obj, SKIP_SIGNAL, True)
+            setattr(obj, OP_LOG_SKIP_SIGNAL, True)
     except:
         pass
 
```

Comment:
There is no need to review this code changes since it does not contain any known irregularities, potential issues, or optimization suggestions for improvement. It appears to be well-written and functional with respect to its current implementation.